Source/Fuse.Drawing.Primitives/Circle.uno

```diff
@@ -96,8 +97,9 @@ namespace Fuse.Drawing.Primitives
 				VertexCount: _bufferIndex.Count();
 				
 				float2 VertexPosition: V0 * (radius + extend*2);
+				float2 VertexPositionScaled: (V0 * (radius + extend*2)) / radius;
```

Comment:
I know I suggested this, but maybe `float2 VertexPositionScaled: VertexPosition / radius;` would be a bit clearer?